markdown/ch3.markdown

```diff
@@ -270,7 +263,7 @@ shows a different feature of the signal.
 Now, think of what happens when the kernel is (1, 0, -1), the difference
 filter, for a signal `s`. At any position `i`, the convolution result is
 `1*s[i+1] + 0*s[i] - 1*s[i-1]`, that is, `s[i+1] - s[i-1]`.
```

Comment:
@stefanv this is a conscious decision to stray from it, first because I think PEP8 makes a bit less sense with inline text, and second to better show pairing of signal with filter values.